scripts/info.js

```diff
@@ -0,0 +1,24 @@
+var information = [];
+
+function Info (catagories) {
+  this.infoTitle = catagories.title; //Title of info
+  this.summery = catagories.summery; //Summery of info
+}
+
+Info.prototype.toHtml = function() {
+  var $infoChunk = $('section').clone();
+  $infoChunk.find('title').html(this.infoTitle);
+  $infoChunk.find('.blogSummery').html(this.summery);
+  $infoChunk.removeAttr('class');
+  console.log('Prototyping: '+this.infoTitle);
+  return $infoChunk;
+};
+
+soccerBlog.forEach(function(element){
+  information.push(new Info(element));
+});
+
+information.forEach(function(soccerPost){ // Get it?   Soccer... Post...
+  $('section').append(soccerPost.toHtml());
+  console.log('appending story: '+soccerPost.this.infoTitle);
```

Comment:
An error is caused by this console.log() because of your use of 'this' inside forEach(). The documentation for .forEach() has info about explicitly passing in an object to serve as 'this' as the method's second parameter. Ultimately, I don't think you need 'this' for console.log to print out what you want to see, try just removing it.